allensdk/internal/api/behavior_data_lims_api.py

```diff
@@ -183,8 +182,10 @@ def get_rewards(self) -> pd.DataFrame:
             delivered rewards.
         """
         data = self._behavior_stimulus_file()
-        # No sync timestamps to rebase on, so pass dummy rebase function
-        return get_rewards(data, lambda x: x)
+        offset = frame_time_offset(data)
+        # No sync timestamps to rebase on, but do need to align to
+        # trial events, so add the offset as the "rebase" function
```

Comment:
Maybe this comment would be better in the `get_rewards` docstring since it looks like the docstring should be updated anyways?

Reply:
I'd like to table this and make another issue for getting some of the internal documentation I linked on Confluence into the AllenSDK so it's clearer.

Reply:
Sounds good to me.